### PR TITLE
AT_017.001 | Pipeline> check icon after build

### PR DESCRIPTION
**Pull request by VitalyVZH**

### CODE REVIEW

**Comment by VitalyVZH** [comment]
[TC_017.001 | Pipeline> check icon after build](https://trello.com/c/HD5FANNc/79-tc017001-pipeline-check-icon-after-build)

**Comment by VitalyVZH** [comment]
https://trello.com/c/HD5FANNc/79-tc017001-pipeline-check-icon-after-build[](url)

**Comment by VitalyVZH** [comment]
> 1

**Comment by VitalyVZH** [comment]
> 1

**Review by RuslanG13 — APPROVED** [review]

**Comment by VitalyVZH** [comment]
[AT_017.001 | Pipeline> check icon after build](https://trello.com/c/UmHMufnZ/588-at017001-pipeline-check-icon-after-build)

File: src/test/java/CreatePipelineCheckStatusIconAfterBuildTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    public void testCheckIcon() {
+        $("[title='New Item']").click();
+        $("#name").sendKeys(NAME);
+        $x("//li[contains(@class,'WorkflowJob')]").click();
+        $("#ok-button").click();
+        $("#cb16 ~ label").click();
+        new Select($(".samples select")).selectByValue("hello");
+        $("[type='submit']").click();
+        $("#jenkins-home-link").click();
+        $x(String.format("//span[contains(text(), '%s')]/following-sibling::*[name()='svg']", NAME)).click();
```
Comment (RusAli) [inline src/test/java/CreatePipelineCheckStatusIconAfterBuildTest.java:23]: I have no idea how we could review that)))

**Review by RusAli — APPROVED** [review]

**Pull request merged**